src/main/java/org/opentripplanner/api/model/ApiLeg.java

```diff
@@ -240,35 +238,4 @@
     @XmlAttribute
     @JsonSerialize
     public Boolean rentedBike;
-
-    /**
-     * Whether this leg is a transit leg or not.
-     * @return Boolean true if the leg is a transit leg
-     */
-    public Boolean isTransitLeg() {
-        if (mode == null) return null;
-        else if (mode.equals(TraverseMode.WALK.toString())) return false;
-        else if (mode.equals(TraverseMode.CAR.toString())) return false;
-        else if (mode.equals(TraverseMode.BICYCLE.toString())) return false;
-        else return true;
-    }
-
-    /**
-     * The leg's duration in seconds
-     */
-    @XmlElement
-    @JsonSerialize
-    public double getDuration() {
-        return endTime.getTimeInMillis()/1000.0 - startTime.getTimeInMillis()/1000.0;
-    }
-
```

Comment:
Why is these methods removed? They are part of the API.

Reply:
Added them back